meshio/vtu/_vtu.py

```diff
@@ -53,6 +54,93 @@ def _cells_from_data(connectivity, offsets, types, cell_data_raw):
         if meshio_type == "polygon" or meshio_type == "polyhedron":
```

Comment:
Can we change this to `meshio_type in ["polygon", "polyhedron"]`? (I know you didn't touch this, it just caught my attention.)

Reply:
Will do.